Document/0x06b-Basic-Security-Testing.md

```diff
@@ -114,37 +114,39 @@ The iOS jailbreak scene evolves so rapidly that providing up-to-date instruction
 
 #### Recommended Tools - iOS Device
 
-Many tools on a jailbroken device can be installed by using Cydia, which is the unofficial AppStore for iOS devices and allows you to manage repositories. One of the most popular repositories is BigBoss, which contains various packages, such as the BigBoss Recommended Tools package. If your Cydia installation isn't pre-configured with this repository, you can add it by navigating to Sources -> Edit, then clicking "Add" in the top left and entering the following URL <http://apt.thebigboss.org/repofiles/cydia/>.
+Many tools on a jailbroken device can be installed by using Cydia, which is the unofficial AppStore for iOS devices and allows you to manage repositories. In Cydia you should add (if not already done by default) the following repositories by navigating to Sources -> Edit, then clicking "Add" in the top left:
 
-Another repository, with quiet a few good tools, is Elucubratus, which gets installed when you install Cydia on iOS 12 using Unc0ver: <https://apt.bingner.com/>.
+- One of the most popular repositories is BigBoss, which contains various packages, such as the BigBoss Recommended Tools package: <http://apt.thebigboss.org/repofiles/cydia/>.
```

Comment:
Maybe list the repo first each time, followed by the description, just like we do for other lists of tools.